ask-smapi-sdk/tests/unit/test_util.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights
+# Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the
+# License.
+#
+
+from ask_smapi_sdk import get_header_value
+
+
+def test_get_header_value_null_header_list():
+    result = get_header_value(header_list=None, key='test_key')
+    assert result == [], ("get_header_value did not return empty list "
```

Comment:
nit: Can we use the [`unittest` framework](https://docs.python.org/3/library/unittest.html) instead, for these tests?